src/components/Books.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Books = (props) => {
+  const { title, author } = props;
+  return (
+    <>
+      <div className="book-container">
+        <ul>
+          <li className="the-book">
+            <span>
+              {title}
+              {' '}
+              Dune
+            </span>
+            <span>
+              {author}
+              {' '}
+              Frank Herbert
+            </span>
+            <button type="submit">Remove</button>
+          </li>
+        </ul>
+        <div className="form-sec">
+          <h2>Add New Book</h2>
+          <form className="form">
+            <div>
+              <input type="text" id="title1" className="title-input" placeholder="Book Title" />
+            </div>
+            <div className="book-type">
+              <select className="title-input">
+                <option value="Action">Action</option>
+                <option value="Science Fiction">Science Fiction</option>
+                <option value="Economy">Economy</option>
+              </select>
+            </div>
+            <div className="submit-btn">
+              <button type="submit">Add book</button>
+            </div>
+          </form>
+        </div>
```

Comment:
- [ ] @george827, React was built on the concept to break pages into components. The more you break the page into small components, the more it becomes easier to use and scalable. This should have been another component called createNewBook. This component should render the form and should have the submit button. Let's kindly rectify this issue.


- [ ] You'll also need the `BookList` that does the following:
- Renders a list of books using the `Book` component
- Renders a form by using the CreateNewBook component